config/sync/recipe_theme.settings.yml

```diff
@@ -0,0 +1,12 @@
+features:
+  node_user_picture: true
+  comment_user_picture: true
+  comment_user_verification: true
+  favicon: true
+logo:
+  use_default: false
+  path: 'public://logo_1_0.png'
```

Comment:
Unfortunately this won't be good. This means that whenever this configuration is imported it expects a file with this exact name to be uploaded in the system - files, when uploaded act similarly to content, they cannot be moved between environments.

Reply:
The info.yml file looks good, but I think this path should be removed. You can edit this on the `/admin/appearance/settings/recipe_theme` page, by unchecking the "Use the logo supplied by the theme" checkbox.

Reply:
It is already unchecked. If I delete the path manually in the file and save, it appears again automatically so git doesn't recognize any changes.